app/views/index.scala.html

```diff
@@ -24,6 +28,15 @@
             background-color: rgba(192, 190, 207, 0.4);
             left: 0px;">
         </div>
+
+@*
```

Comment:
I'm assuming that this is an extra commented out line for testing. Can you remove this since we won't want it merged?